assets/js/src/store.js

```diff
@@ -0,0 +1,26 @@
+import appReducer from "./containers/App/reducer";
+import loginReducer from "./containers/Login/reducer";
+import { createBrowserHistory } from "history";
+import { applyMiddleware, combineReducers, compose, createStore } from "redux";
+import { connectRouter, routerMiddleware } from "connected-react-router";
+
+// get browser history
+export const history = createBrowserHistory();
+
+// combine reducers from all containers
+const rootReducer = combineReducers({
+  app: appReducer,
+  login: loginReducer
+});
+
+// connect routing to store to allow time-travel/etc. between routes
+const reducerWithRouting = connectRouter(history)(rootReducer);
+
+// supprt redux devtools extension
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
This allows the Redux DevTools Extension to work, you should install it, it's very useful 🙂 
chrome https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en
firefox https://addons.mozilla.org/en-US/firefox/addon/remotedev/